Makefile

```diff
@@ -24,19 +24,21 @@ BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
 endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
-# IMAGE_TAG_BASE defines the docker.io namespace and part of the image name for remote images.
-# This variable is used to construct full image tags for bundle and catalog images.
-#
-# For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
-# openshift.io/recipe-bundle:$VERSION and openshift.io/recipe-catalog:$VERSION.
-IMAGE_TAG_BASE ?= openshift.io/recipe
+# Variables for the container image name
+IMAGE_REGISTRY ?= quay.io
+IMAGE_REPOSITORY ?= ramendr
+IMAGE_NAME ?= recipe
+IMAGE_TAG_BASE = $(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME)
```

Comment:
This is not a tag, maybe `prefix` since it is used as a prefix for IMG, BUNDLE_IMG, and CRD_BUNDLE_IMG.

Using lower case to make it clear that this is an internal variable and not something that the user should override.

Or just get rid of this tempoary variable and use the IMAGE_* variables when in the 3 cases using IMAGE_TAG_BASE.

Reply:
Done